pb/edge_client_pb/edge_client.proto

```diff
@@ -135,8 +136,13 @@ message PostureResponse {
     Woken woken = 5;
     Unlocked unlocked = 6;
     SdkInfo sdkInfo = 7;
+    TotpToken totpToken = 8;
   };
 
```

Comment:
What's the advantage of wrapping this in a type, instead of just using string?

Reply:
I was unsure if other fields would be necessary in the future. I could use a string here, but if a field is added later, it would be backward-incompatible or I would have to add a different but similarly named field.